src/core/core.datasetController.js

```diff
@@ -319,7 +319,9 @@ helpers.extend(DatasetController.prototype, {
 	 * @private
 	 */
 	onDataPush: function() {
-		this.insertElements(this.getDataset().data.length - 1, arguments.length);
+		var me = this;
```

Comment:
[nit] I think this actually increases the size of this function during minification. This line would get minified to something like `var x=this;` which is 11 bytes long. We use `me` twice in this method which adds 2 more bytes for a total of 13. However we replaced `this` twice which subtracts 8 bytes for a gain of 5.

If my math is correct we should only do this when `this` gets removed 4 or more times

Reply:
Removed that useless cache variable.